src/test/java/org/jboss/xavier/integrations/jpa/service/UserServiceTest.java

```diff
@@ -38,4 +38,10 @@ public void userService_AnalysisGiven_ShouldReturnUser() {
         assertThat(userService.findUser("mrizzi@redhat.com").isFirstTimeCreatingReports()).isEqualTo(false);
         assertThat(userService.findUser("whatever").isFirstTimeCreatingReports()).isEqualTo(true);
     }
+
+    @Test
+    public void userService_AuthorizedUsersEmpty_ShouldReturnNotAllowed() {
+        ReflectionTestUtils.setField(userService, "authorizedAdminUsers", new String[0]);
+        assertThat(userService.isUserAllowedToAdministratorResources("myUsername")).isEqualTo(false);
+    }
```

Comment:
Better add a test to cover the case when `rest.authorization.administration` has values
```suggestion
    }
    
    @Test
    public void userService_AuthorizedUsersGiven_ShouldReturnAllowedOrNot() {
        assertThat(userService.isUserAllowedToAdministratorResources("admin1")).isEqualTo(true);
        assertThat(userService.isUserAllowedToAdministratorResources("admin1@redhat.com")).isEqualTo(false);
        assertThat(userService.isUserAllowedToAdministratorResources("admin2@redhat.com")).isEqualTo(true);
    }
```